src/components/modals/Agenda/Agenda.tsx

```diff
@@ -60,8 +71,38 @@ export default function Agenda({
             >
                 Disse timene vil bli booket automatisk
             </Typography>
+            {userConfig?.active === false && (
+                <Alert severity={"info"} icon={<PauseCircleRounded />}>
+                    <AlertTitle>Automatisk booking er satt på pause</AlertTitle>
+                    Du kan skru på automatisk booking i innstillinger, slik at timene i timeplanen blir booket
+                    automatisk
+                </Alert>
+            )}
+            {hasGhostClasses && userConfig?.active && (
+                <Alert
+                    severity={"error"}
+                    icon={
+                        <Avatar
+                            alt={"Ghost class"}
+                            src={"/ghost.png"}
+                            sx={{
+                                width: 20,
+                                height: 20,
+                            }}
+                        />
+                    }
+                >
+                    <AlertTitle>Utdatert timeplan</AlertTitle>
+                    En eller flere av timene i timeplanen din er ikke satt opp denne uka. Kontroller at timeplanen din
+                    stemmer overens med ukas gruppetimer.
```

Comment:
```suggestion
                    En eller flere av timene i planen din går ikke denne uka. Kontroller at planen din
                    stemmer overens med ukas gruppetimer.
```

Reply:
I feel like that formulation ("går ikke") could be interpreted as "not possible this week" instead of saying that it has not been scheduled. 🤔

Reply:
Ehh, I kinda agree after trying on the different texts. One should understand it based on the context, since you are in a "timeplan" window. Fixed in https://github.com/mathiazom/rezervo-web/pull/100/commits/0d923cf38a187e399439e555e661827410c2b261